apps/200-flexdashboard-render-text/index.Rmd

```diff
@@ -62,55 +50,98 @@ shinyjster::shinyjster_js("
     })
   }
 
-  assert_equal_qs('a[href=\"#section-p1\"]', 'Page 1: Feb');
-  assert_equal_qs('a[href=\"#section-p2\"]', 'Page 2: Feb');
-  assert_equal('sidebarContent', 'Sidebar Content: Feb');
-
-  assert_equal_qs('#section-hidden1 .value-output', 'VB1: Feb');
-  assert_equal_qs('#section-hidden2 .value-output', 'VB2: Feb');
-  assert_equal_qs('#section-hidden3 .value-output', 'VB3: Feb');
+  function assert_page(idx, name) {
```

Comment:
Please update the "name" variable to be "month" (through the shinyjster app) for better readability. Thank you.